src/test/java/org/jscep/server/ScepServletTest.java

```diff
@@ -357,6 +369,99 @@ public void testRenewal() throws Exception {
         assertThat(renewalSate, is(State.CERT_ISSUED));
     }
 
+    @Test
+    public void testMissingOperation() throws Exception {
+        Map<String, String> expectedResponseHeaders = new HashMap<String, String>();
+        expectedResponseHeaders.put("Cache-Control", "must-revalidate,no-cache,no-store");
+        expectedResponseHeaders.put("Content-Type", "text/html;charset=ISO-8859-1");
+        expectedResponseHeaders.put("Content-Length", "1327");
+        expectedResponseHeaders.put("Server", "Jetty(7.6.4.v20120524)");
+
+        String expectedResponseBody =
```

Comment:
I think checking that the response contains the string would be more resilient to Jetty changes.

Reply:
Agreed.  Another force-push, but I think now each commit shows how the responses transition with each code change.